trade-finance-manager-api/src/v1/api.js

```diff
@@ -340,25 +340,22 @@ const updateFacility = async ({ facilityId, tfmUpdate, auditDetails }) => {
 };
 
 const createFacilityAmendment = async (facilityId, auditDetails) => {
-  const isValid = isValidMongoId(facilityId) && hasValidUri(DTFS_CENTRAL_API_URL);
-  if (isValid) {
-    try {
-      const response = await axios({
-        method: 'post',
-        url: `${DTFS_CENTRAL_API_URL}/v1/tfm/facilities/${facilityId}/amendments`,
-        headers: headers.central,
-        data: { auditDetails },
-      });
+  if (!isValidMongoId(facilityId)) {
+    throw new InvalidFacilityIdError(facilityId);
+  }
 
-      return response.data;
-    } catch (error) {
-      console.error('Error creating facility amendment %o', error);
-      return { status: error?.response?.status || 500, data: 'Failed to create facility amendment' };
-    }
-  } else {
-    console.error('Invalid facilityId provided');
-    return { status: 400, data: 'Invalid facility id' };
+  if (!hasValidUri(DTFS_CENTRAL_API_URL)) {
```

Comment:
Can this be applied to other endpoints as well please, appreciate a new PR will be required for this.

Reply:
I'll raise another PR for this, but this validation should take place on application startup, not every time we hit an end point as its a static constant :)